cardano-node/src/Cardano/Node/Tracing/Tracers/Startup.hs

```diff
@@ -0,0 +1,414 @@
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE UndecidableInstances #-}
+{-# OPTIONS_GHC -Wno-name-shadowing -Wno-orphans #-}
```

Comment:
missing newline between language pragmas and module name

Reply:
Done